testsuite/src/test/resources/parse-serial-gc-build-and-run.log

```diff
@@ -0,0 +1,1066 @@
+[INFO] 
+[INFO] --- quarkus-maven-plugin:2.13.4.Final:build (default) @ quarkus-json ---
+[INFO] [io.quarkus.deployment.pkg.steps.JarResultBuildStep] Building native image source jar: /home/karm/workspaceRH/mandrel-integration-tests-dev/apps/quarkus-json/target/quarkus-json_-ParseOnce-native-image-source-jar/quarkus-json_-ParseOnce-runner.jar
+[INFO] [io.quarkus.deployment.pkg.steps.NativeImageBuildStep] Building native image from /home/karm/workspaceRH/mandrel-integration-tests-dev/apps/quarkus-json/target/quarkus-json_-ParseOnce-native-image-source-jar/quarkus-json_-ParseOnce-runner.jar
+[INFO] [io.quarkus.deployment.pkg.steps.NativeImageBuildStep] Running Quarkus native-image plugin on GraalVM 23.0.0-dev Java 17 CE (Java Version 17.0.5+6-jvmci-23.0-b01)
+[INFO] [io.quarkus.deployment.pkg.steps.NativeImageBuildRunner] /home/karm/X/JDKs/graalvm-ce-java17-23.0.0-dev/bin/native-image -J-Djava.util.logging.manager=org.jboss.logmanager.LogManager -J-Dsun.nio.ch.maxUpdateArraySize=100 -J-Dlogging.initial-configurator.min-level=500 -J-Dvertx.logger-delegate-factory-class-name=io.quarkus.vertx.core.runtime.VertxLogDelegateFactory -J-Dvertx.disableDnsResolver=true -J-Dio.netty.leakDetection.level=DISABLED -J-Dio.netty.allocator.maxOrder=3 -J-Duser.language=en -J-Duser.country=US -J-Dfile.encoding=UTF-8 --features=io.quarkus.runner.Feature,io.quarkus.runtime.graal.ResourcesFeature,io.quarkus.runtime.graal.DisableLoggingFeature -J--add-exports=java.security.jgss/sun.security.krb5=ALL-UNNAMED -J--add-opens=java.base/java.text=ALL-UNNAMED -J--add-opens=java.base/java.io=ALL-UNNAMED -J--add-opens=java.base/java.lang.invoke=ALL-UNNAMED -J--add-opens=java.base/java.util=ALL-UNNAMED -H:+CollectImageBuildStatistics -H:ImageBuildStatisticsFile=quarkus-json_-ParseOnce-runner-timing-stats.json -H:BuildOutputJSONFile=quarkus-json_-ParseOnce-runner-build-output-stats.json -H:+AllowDeprecatedBuilderClassesOnImageClasspath -R:MaxHeapSize=2560m -H:-ParseOnce -H:BuildOutputJSONFile=quarkus-json_minus-ParseOnce.json -H:InitialCollectionPolicy=com.oracle.svm.core.genscavenge.CollectionPolicy\$BySpaceAndTime -H:+AllowFoldMethods -J-Djava.awt.headless=true --no-fallback --link-at-build-time -H:+ReportExceptionStackTraces -H:-AddAllCharsets --enable-url-protocols=http -H:NativeLinkerOption=-no-pie -H:-UseServiceLoaderFeature -H:+StackTrace -J--add-exports=org.graalvm.sdk/org.graalvm.nativeimage.impl=ALL-UNNAMED -J--add-exports=org.graalvm.nativeimage.builder/com.oracle.svm.core.jdk=ALL-UNNAMED quarkus-json_-ParseOnce-runner -jar quarkus-json_-ParseOnce-runner.jar
+Warning: Class-path entry file:///home/karm/workspaceRH/mandrel-integration-tests-dev/apps/quarkus-json/target/quarkus-json_-ParseOnce-native-image-source-jar/lib/org.graalvm.sdk.graal-sdk-22.3.0.jar contains class com.oracle.svm.core.annotate.TargetElement. This class is part of the image builder itself (in jrt:/org.graalvm.sdk) and must not be passed via -cp.
```

Comment:
What does it take to get rid of these warnings ?

Reply:
I think we need to address that on the Quarkus side. It is what you get when running Q2.13 with the GraalVM 23 dev.

IMHO:  
 * https://github.com/oracle/graal/commit/90c1f568d629579abee670d369faf91e0847df6c
 * https://github.com/oracle/graal/blob/b6e1c8d0844c475c97ad7ef52796b192b59ef2fc/substratevm/CHANGELOG.md?plain=1#L6

Reply:
> I think we need to address that on the Quarkus side.

I have actually fixed this already... https://github.com/quarkusio/quarkus/pull/28650

I will need to start doing some memory exercises...

Reply:
THX :pray: :+1: